frontend/src/components/Table/ViewPropertyCell.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { CellProps } from 'react-table';
+import { IProperty } from 'actions/parcelsActions';
+import { Link } from 'react-router-dom';
+import queryString from 'query-string';
+import { PropertyTypes } from 'constants/propertyTypes';
+
+/**
+ * A cell that provides a clickable link to view a given IProperty
+ * @param {CellProps<IProperty, number>} props
+ */
+const ViewPropertyCell = (props: CellProps<IProperty, number>) => {
```

Comment:
moving this to its own component to promote reuse and consistency across tables.

Reply:
also makes the code more testible, can now test this code in isolation without needing a table.